spec/requests/notes_spec.rb

```diff
@@ -19,29 +19,67 @@
       get notes_url
       expect(response).to be_successful
     end
+
```

Comment:
But again, this would require us to duplicate parts of the set-up logic which I'm not a friend of ... Or would it be a common practice in Ruby to try to express the test in multiple partial classes (I have no idea how this would be possible at all)?

Reply:
Maybe that's possible but it sounds like a little bit too much overhead :sweat_smile: 
Lucky for you that Christoph 'fixed' this issue by adding

```ruby

# Ignore "Block has too many lines" for specs, `RSpec.describe` blocks can be long
Metrics/BlockLength:
  Exclude:
    - 'spec/**/*.rb'
```

to the rubocop config :D